### PR TITLE
 Improve: indent rhs of ref update

### PR DESCRIPTION
**Pull request by hhugo**

### CODE REVIEW

File: src/Fmt_ast.ml [hunk 1]
```diff
@@ -1010,6 +1010,12 @@ and fmt_expression c ?(box= true) ?epi ?eol ?parens ?ext
         ( fmt_expression c (sub_exp ~ctx s) $ fmt ".["
         $ fmt_expression c (sub_exp ~ctx i) $ fmt "]@ <- "
         $ fmt_expression c (sub_exp ~ctx e) )
+  | Pexp_apply
+      ( {pexp_desc= Pexp_ident {txt= Lident ":="}; pexp_attributes= []}
+      , [(Nolabel, r); (Nolabel, v)] ) ->
```
Comment (jberdine) [inline src/Fmt_ast.ml]: You probably need to add `hovbox 0` here analogous to the general case below, or else get cases like
```
;; if cccccccccccccccccccc then x :=
       f aaaaaaaaaaaaaaaaaaaa bbbbbbbbbbbbbbbbbbbb
```

Comment (hhugo) [reply src/Fmt_ast.ml]: indeed

File: test/passing/refs.ml [hunk 2]
```diff
@@ -5,6 +5,10 @@ let _ =
   xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx :=
     something very
       looooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooong ;
+  xxxxxxxxxxxxx xxxxxxxxxxx xxxxxxxxxxxx xxxxxxxxx xxxxxxxxxxxxx xxxxxxxxxx
+    xxxxxxxxxxxxx :=
+    something very
+      looooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooong ;
```
Comment (jberdine) [inline test/passing/refs.ml]: @hhugo what do you think of this one? To me this would be clearer with `:=` on the left margin.

Comment (hhugo) [reply test/passing/refs.ml]: I'm with you, the following look better to me. @mbarbin, any feedback ?
```
xxxxxxxxxxxxx xxxxxxxxxxx xxxxxxxxxxxx xxxxxxxxx xxxxxxxxxxxxx xxxxxxxxxx
  xxxxxxxxxxxxx
:=
  something very
    looooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooong ;
```

Comment (mbarbin) [reply test/passing/refs.ml]: +1.  I find it is easier to spot the operator `:=` when it is on the left margin.

Comment (jberdine) [reply test/passing/refs.ml]: Ok, it sounds like using the previous formatting in case `is_simple` does not hold of the lhs might work.

**Comment by hhugo** [comment]
now only applying the style when lhs is simple.

**Comment by hhugo** [comment]
Thanks

**Comment by jberdine** [comment]
You did the work!

**Comment by hhugo** [comment]
You've pressed the merge button :)

**Pull request merged**